javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/PropagatorsInitializer.java

```diff
@@ -82,73 +102,20 @@ public static void initializePropagators(List<String> propagatorIds) {
       }
     }
     if (propagators.size() > 1) {
-      textMapPropagators.add(new MultiPropagator(propagators));
+      textMapPropagators.addAll(propagators);
     } else if (propagators.size() == 1) {
       textMapPropagators.add(propagators.get(0));
     }
```

Comment:
Can't it be replaced by just `textMapPropagators.addAll(propagators);`? Or you can add directly to the `textMapPropagators` list, without using the `propagators` one

Reply:
Great call!  Originally this size check was there as an optimization to prevent unnecessary compositing.  That logic also exists in `TextMapPropagator.composite()`, so it doesn't need to be here.  Since the tests still pass, I'll make this change...thanks!